pymc3/smc/smc.py

```diff
@@ -341,6 +337,7 @@ def __init__(
         self.observations = self.sum_stat(observations)
 
     def posterior_to_function(self, posterior):
+        model = self.model
```

Comment:
Sorry, why is this added here if it's unused?

Reply:
Good catch! This is a mistake, this is part of other ideas I was trying that did not make it into the PR.

Reply:
@aloctavodia while we're here, just wanted to say that I'm studying your book and that it's really good!

Reply:
Glad you like it. @MarcoGorelli. Hopefully, next year we will publish a new one together with Junpeng, Ravin and Ari

Reply:
Awesome @aloctavodia - feel free to reach out to me if you want any help proof-reading